.github/workflows/e2e.yml

```
@@ -148,7 +148,7 @@ jobs:
     strategy:
       fail-fast: false # keeps matrix running if one fails
       matrix:
-        rn-version: ['0.65.3', '0.72.4']
+        rn-version: ['0.65.3', '0.73.0-rc.3']
```

Since this is going to main, shouldn't we use `0.73.0` instead of `0.73.0-rc.3` ?
```suggestion
        rn-version: ['0.65.3', '0.73.0']
```